metricbeat/module/kibana/stats/_meta/data.json

```diff
@@ -4,45 +4,52 @@
         "hostname": "host.example.com",
         "name": "host.example.com"
     },
-    "elasticsearch": {
-        "cluster": {
-            "id": "njqU4EQaTROIDlWPeUMQyw"
-        }
-    },
     "kibana": {
         "stats": {
-            "concurrent_connections": 0,
+            "concurrent_connections": 3,
+            "elasticsearch": {
```

Comment:
This should be `elasticsearch.cluster.id` on the Root level. Same for `process.pid`

Reply:
Ah doh. Fixing!